core/test/libraries/Volatility.t.sol

```diff
@@ -38,6 +38,23 @@ contract VolatilityTest is Test {
         );
         assertEq(dailyIV, 20405953567249984); // 2.041%
 
+        dailyIV = Volatility.estimate(
+            metadata,
+            data,
+            Volatility.FeeGrowthGlobals(
+                1501955347902231987349614320458936,
+                527278396421895291380335427321388844898052,
+                0
+            ),
+            Volatility.FeeGrowthGlobals(
+                1501968291161650295867029090958139,
+                527315901327546020416261134123578344760082,
+                8640
+            ),
+            1 hours
+        );
+        assertEq(dailyIV, 4165347829168579); // 0.417%
+
```

Comment:
Good - example of volatility estimate that reverted in the old code, but no longer reverts now